### PR TITLE
Add Mention of ZHA Group Entities to Release Notes

### PR DESCRIPTION
**Pull request by tube0013**
## Proposed change

This highlights the new entities created by zigbee groups in zha, which was a huge desire for folks after zigbee groups were added

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_posts/2020-04-08-release-108.markdown [hunk 1]
```diff
@@ -156,6 +156,11 @@ Before and after screenshot for the Horizontal Stack improvements.
 
 - The markdown card has a new `user` variable that can be used to display the
   currently logged in user.
+  
+- With the ZHA ZigBee integration, ZigBee groups now dynamically create 
+  entities for lights, switches and fans that are grouped together. This 
+  enables near simultaneous control over a large group of lights allowing 
+  them to turn on or off at the exact same time. Thank you [@dmulcahey]!
```
Comment (frenck) [inline source/_posts/2020-04-08-release-108.markdown]:
```suggestion
- With the ZHA Zigbee integration, Zigbee groups now dynamically create 
  entities for lights, switches and fans that are grouped together. This 
  enables near-simultaneous control over a large group of lights allowing
  them to turn on or off at the exact same time. Thank you [@dmulcahey]!
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks, @tube0013!

Left a small suggestion, could you take a look? Thanks!

**Comment by tube0013** [comment]
Thanks!

**Review by frenck — APPROVED** [review]
No! Thanks you @tube0013 👍

**Pull request merged**